demos/browser/app/meetingReadinessChecker/meetingReadinessChecker.html

```diff
@@ -50,6 +50,8 @@ <h1 class="h3 mb-3 font-weight-normal">Amazon Chime SDK Readiness Checker</h1>
           <option value="us-east-2">United States (Ohio)</option>
           <option value="us-west-1">United States (N. California)</option>
           <option value="us-west-2">United States (Oregon)</option>
+          <option value="us-gov-west-1">GovCloud (US-West)</option>
+          <option value="us-gov-east-1">GovCloud (US-East)</option>
```

Comment:
While it may be good to add these regions for the readiness checker, I don't think these changes are needed to support live transcription.

Reply:
will remove these lines.